src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -310,7 +308,7 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
     }
 
     const loadBucketLocation = async (googleProject, bucketName) => {
-      if (!!googleProject) {
+      if (isGoogleWorkspace && workspace) {
```

Comment:
I'm surprised this isn't showing as a merge conflict. @wnojopra just fixed a bug here in https://github.com/DataBiosphere/terra-ui/pull/3115/files

Reply:
Looks like [this PR's branch](https://github.com/DataBiosphere/terra-ui/commits/jc-run-analysis) contains the commit from #3115, so I'm guessing this was manually resolved at some point. In which case, this looks like a regression.